Content.Server/Materials/MaterialStorageSystem.cs

```diff
@@ -93,28 +93,41 @@ public override bool TryInsertMaterialEntity(EntityUid user,
         EntityUid receiver,
         MaterialStorageComponent? storage = null,
         MaterialComponent? material = null,
-        PhysicalCompositionComponent? composition = null)
+        PhysicalCompositionComponent? composition = null,
+        bool trySplitStacks = false)
     {
         if (!Resolve(receiver, ref storage) || !Resolve(toInsert, ref material, ref composition, false))
             return false;
         if (TryComp<ApcPowerReceiverComponent>(receiver, out var power) && !power.Powered)
```

Comment:
This is the only thing keeping this method from being moved to shared, which can be solved with an API call.
```suggestion
        if (!SharedPowerReceiverSystem.IsPowered(receiver))
```